### PR TITLE
DOC: Cleanups to roadmap, governance, BIAPs

### PR DESCRIPTION
**Pull request by effigies**
Remaining issues:

* ~~``:ref:`contributor_guide` `` does not exist~~
* ~~``:ref:`core_dev` `` does not exist~~
* ~~Don't understand this line (Xarray is not mentioned prominently in napari docs):~~
  > Xarray simplifies HDF5 serialization, and visualization; see https://napari.org.

### CODE REVIEW

**Comment by matthew-brett** [comment]
Sorry - I guess `contributor_guide` should be`devguide`.

`core_dev` is another networkx page: <https://github.com/networkx/networkx/blob/main/doc/developer/core_developer.rst>, I guess we could just make our own copy and add that.

Maybe just drop the reference to Napari in the offending sentence?

**Comment by effigies** [comment]
I think this should be ready for review.

File: doc/source/devel/core_developer.rst [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+   example that not only illustrates but explains it.
+
+3. **The algorithm:** You should understand the code being modified or
+   added before approving it.  (See `Merge Only Changes You
+   Understand`_ below.) Implementations should do what they claim,
+   and be simple, readable, and efficient.
+
+4. **Tests:** All contributions to the library *must* be tested, and
+   each added line of code should be covered by at least one test. Good
+   tests not only execute the code, but explores corner cases.  It is tempting
```
Comment (matthew-brett) [inline doc/source/devel/core_developer.rst:51]:
```suggestion
   tests not only execute the code, but explore corner cases.  It is tempting
```

File: doc/source/devel/core_developer.rst [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+make the changes by `pushing to their branch
+<https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork>`__,
+or using GitHub’s `suggestion
+<https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request>`__
+`feature
+<https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request>`__.
+(The latter is preferred because it gives the contributor a choice in
+whether to accept the changes.)
+
+Our default merge policy is to squash all PR commits into a single
```
Comment (matthew-brett) [inline doc/source/devel/core_developer.rst:65]: I think we aren't doing that at the moment.  Do we want to?   I personally prefer leaving the commits as they are.

File: doc/source/devel/core_developer.rst [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+among others. Similarly, pull requests sometimes need to be closed without
+merging, because:
+
+- the pull request implements a niche feature that we consider not worth the
+  added maintenance burden;
+- the pull request implements a useful feature, but requires significant
+  effort to bring up to Nibabel's standards, and the original
+  contributor has moved on, and no other developer can be found to make the
+  necessary changes; or
+- the pull request makes changes that do not align with our values, such as
```
Comment (matthew-brett) [inline doc/source/devel/core_developer.rst:122]:
```suggestion
- the pull request makes changes that make maintenance harder, such as
```

File: doc/source/devel/governance.rst [hunk 4]
```diff
@@ -40,7 +40,7 @@ project in concrete ways, such as:
 among other possibilities. By contributing to the project, community members
 can directly help to shape its future.
 
-Contributors should read the :ref:`contributor_guide` and our :ref:`code_of_conduct`.
+Contributors should read the :ref:`chap_devguide` and our :ref:`code_of_conduct`.
```
Comment (matthew-brett) [inline doc/source/devel/governance.rst:43]: Do we have a `code_of_conduct` ref?   As you know, I personally prefer 'community guidelines' as a document title.

File: doc/source/devel/core_developer.rst [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+keep contributing, regardless of the outcome of past contributions.
+
+Further resources
+-----------------
+
+As a core member, you should be familiar with community and developer
+resources such as:
+
+-  Our :ref:`chap_devguide`
+-  Our :ref:`code_of_conduct`
```
Comment (matthew-brett) [inline doc/source/devel/core_developer.rst:149]: Same comment as above - do we have a ref?  Rename?

File: doc/source/devel/core_developer.rst [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+-  `PEP257 <https://www.python.org/dev/peps/pep-0257/>`__ and the `NumPy
+   documentation
+   guide <https://numpy.org/doc/stable/docs/howto_document.html>`__
+   for docstrings. (NumPy docstrings are a superset of PEP257. You
+   should read both.)
+-  The Nibabel `tag on NeuroStars <https://neurostars.org/tag/nibabel>`__
+-  Our `mailing
+   list <https://mail.python.org/mailman/listinfo/neuroimaging>`__
+
+You are not required to monitor all of the social resources.
```
Comment (matthew-brett) [inline doc/source/devel/core_developer.rst:160]:
```suggestion
Please do monitor any of the resources above that you find helpful.
```

**Review by matthew-brett — COMMENTED** [review]
Nice - thanks.  I made a few comments / suggestions.

File: doc/source/devel/core_developer.rst (hunk 2 above)
Comment (effigies) [reply doc/source/devel/core_developer.rst:65]: Ah, right. Meant to add a discussion point on this. Yeah, I'm fine sticking with merge commits, but it means we need to rewrite or remove this whole paragraph.

File: doc/source/devel/governance.rst (hunk 4 above)
Comment (effigies) [reply doc/source/devel/governance.rst:43]: I think you added these in for aliases:

https://github.com/nipy/nibabel/blob/7fc471f8895188497a75914fb137f9a048916128/doc/source/devel/devguide.rst#L114-L121

Can `sed` the refs anyway...

File: doc/source/devel/core_developer.rst (hunk 2 above)
Comment (matthew-brett) [reply doc/source/devel/core_developer.rst:65]: I think removing the paragraph is OK if that's OK with you (I mean - sticking to default previous merge policy).
